cfpb/cfgov-refresh
# Remove BCFP code and images

![Phew! Close call.](https://user-images.githubusercontent.com/1044670/50406520-538fd480-0794-11e9-9266-88c65d3031d2.gif)

![But we made it.](https://user-images.githubusercontent.com/1044670/50406524-6e624900-0794-11e9-8ecb-b3dddeda795c.gif)


## Removals

- `BCFP_LOGO` flag and all uses of it
- BCFP logo images


## Changes

- Reverts some places where we had already changed copy to use BCFP


## Testing

1. Pull branch
1. Browse around the site on your local server and make sure nothing breaks or renders incorrectly, particularly the homepage and the Find a Housing Counselor page
1. Search the codebase for `bcfp` or `bureau of consumer financial protection` and see that the only remaining references are within iRegs (which seem like they should remain)


## Checklist
